packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -1066,6 +1089,14 @@ suite('lit-html', () => {
     test('noChange works on one of multiple expressions', () => {
       const go = (a: any, b: any) =>
         render(html`<div foo="${a}:${b}"></div>`, container);
+
+      go('A', noChange);
```

Comment:
Let's clarity the name of the test, perhaps `noChange renders as empty string when used in interpolated attributes`.